### PR TITLE
INT-2238: Select events to emit

### PR DESCRIPTION
**Pull request by jscasca**

### CODE REVIEW

**Review by SimonFc — APPROVED** [review]
Should we add a test for this?

File: tinymce-angular-component/src/main/ts/utils/Utils.ts [hunk 1]
```diff
@@ -8,15 +8,27 @@
 
 import { EventEmitter } from '@angular/core';
 import { EditorComponent } from '../editor/editor.component';
-import { validEvents } from '../editor/Events';
+import { validEvents, Events } from '../editor/Events';
 
 const bindHandlers = (ctx: EditorComponent, editor: any): void => {
-  validEvents.forEach((eventName) => {
+  const ignoredEvents = parseStringProperty(ctx.ignoreEvents, []);
+  const allowedEvents = parseStringProperty(ctx.allowedEvents, validEvents).filter((event) => !ignoredEvents.includes(event)) as (keyof Events)[];
```
Comment (tiny-james) [inline tinymce-angular-component/src/main/ts/utils/Utils.ts:15]: Is it intended that the user can now add new events that are not in `validEvents`?

**Review by tiny-james — APPROVED** [review]

File: tinymce-angular-component/src/main/ts/utils/Utils.ts (hunk 1 above)
Comment (jscasca) [reply tinymce-angular-component/src/main/ts/utils/Utils.ts]: Filter by valid events now

**Comment by jscasca** [comment]
Added some tests

**Review by SimonFc — APPROVED** [review]

**Review by tiny-james — APPROVED** [review]

**Review by tinydylan — APPROVED** [review]

**Pull request merged**